contentcuration/kolibri_content/migrations/0009_contentnode_options.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2020-04-11 19:07
+from __future__ import unicode_literals
+
+from django.db import migrations
+import jsonfield.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('content', '0008_channelmetadata_tagline'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='contentnode',
+            name='options',
+            field=jsonfield.fields.JSONField(default={}),
```

Comment:
For optimizations, think this needs to be separated into [two migration files](https://www.braintreepayments.com/blog/safe-operations-for-high-volume-postgresql/)